lib/src/dartdoc_options.dart

```diff
@@ -433,6 +433,17 @@ class _YamlFileData {
   _YamlFileData(this.data, this.canonicalDirectoryPath);
 }
 
+/// An enum to specify the multiple different kinds of data an option might
+/// represent.
+enum OptionDataIs {
```

Comment:
`OptionFormat`? `OptionKind`?

Reply:
I think I'll go with `OptionKind`, the only other uses we have of `kind` are pretty far away from here.